src/Components/AnimatedPage.js

```diff
@@ -0,0 +1,22 @@
+import { motion } from "framer-motion";
+
+const AnimatedPage = ({ children }) => {
+  const animations = {
+    initial: { opacity: 0 },
+    animate: { opacity: 1 },
+    exit: { opacity: 0 },
+  };
+  return (
+    <motion.div
+      variants={animations}
+      initial="initial"
+      animate="animate"
+      exit="exit"
+      transition={{ duration: 0.5 }}
+    >
+      {children}
+    </motion.div>
+  );
+};
+
+export default AnimatedPage;
```

Comment:
I see you using the AnimatedPage component in a lot of different child components. Would it be easier to have it in the app.js return so it's always there instead of repeating it?

Reply:
Just gave it a go on App.js instead of the individual components! It didn't work quite as intended, and I think it's because it's designed to fade in and out on a single component. So if I wrap all of my components in the animation the activity inside doesn't trigger anything because there's no "exit" point. I see what you mean though! In the future I think I'll look into refactoring the animation to be applied globally! 😊